src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -15,6 +15,7 @@
     LastUpdatedByName NVARCHAR(256) NULL,     
 
     CONSTRAINT FK_Order_OrderStatusId_OrderStatus_OrderStatusId FOREIGN KEY (OrderStatusId) REFERENCES OrderStatus (OrderStatusId), 
-    CONSTRAINT FK_Order_AddressId_Address_AddressId FOREIGN KEY (OrganisationAddressId) REFERENCES Address (AddressId),
+    CONSTRAINT FK_Order_OrganisationAddressId_Address_AddressId FOREIGN KEY (OrganisationAddressId) REFERENCES Address (AddressId),
+    CONSTRAINT FK_Order_OrganisationBillingAddressId_Address_AddressId FOREIGN KEY (OrganisationBillingAddressId) REFERENCES Address (AddressId),

```

Comment:
Maybe slap some square brackets around Address.

It's not actually a SQL keyword but VS likes to highlight it for some reason...

Reply:
👍